torch/fx/experimental/proxy_tensor.py

```diff
@@ -1070,14 +1130,15 @@ def wrap_fake(x):
         else:
             func = f
 
+        proxy_func_mode = ProxyTorchFunctionMode(fx_tracer)
         # We disable the autocast cache as the autocast cache causes type conversions on parameters to
         # check a cache, which introduces untracked tensors into the graph
         #
         # We also disable tracing by any other tensor proxy-based tracers except the current. The
         # purpose of `make_fx` is to produce graphmodules as a side effect; its internal execution is
         # thus irrelevant to any external functional trace.
         with decompose(decomposition_table), fake_tensor_mode, python_dispatcher_mode, pre_dispatch_mode, proxy_function_mode, \
-             sym_mode, proxy_mode, disable_autocast_cache():
+             sym_mode, proxy_func_mode, proxy_mode, disable_autocast_cache():
```

Comment:
it probably works fine. But could you add test case where we also run strict mode and check this new functionality is not overriding dynamo produced source_fn?